installer/alpine/Dockerfile

```diff
@@ -17,56 +17,54 @@ RUN apk add --update --upgrade \
         openssh \
         squashfs-tools \
         tcpdump \
-        ;
-RUN apk cache sync
-RUN mv /etc/apk/cache /cache
-RUN apk add --no-scripts --no-cache --update --upgrade \
+        && \
+    apk add --no-scripts --update --upgrade --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing \
+        kexec-tools \
+	&& \
+    apk cache sync && \
+    mv /etc/apk/cache /cache && \
+    apk add --no-scripts --no-cache --update --upgrade \
         abuild \
         alpine-sdk \
         build-base \
         busybox-initscripts \
         coreutils \
         linux-headers \
-        ;
-
-# remove edge when linux-vanilla is > 4.14.8 and has the nfp driver enabled,
-# likely alpine v3.8
-RUN apk add --no-scripts --no-cache --update --upgrade --repository http://dl-cdn.alpinelinux.org/alpine/edge/main \
+        && \
+    # remove edge when linux-vanilla is > 4.14.8 and has the nfp driver enabled,
+    # likely alpine v3.8
+    apk add --no-scripts --no-cache --update --upgrade --repository http://dl-cdn.alpinelinux.org/alpine/edge/main \
```

Comment:
Can we see if this is no longer needed? I believe this was to get around some old packages.

Reply:
the comment doesn't make any more sense but we still want latest and greatest linux-firmare. I tried building off of the pinned aports config but it wasn't working for me yesterday. Checked again today and it looks like the dev.alpinelinux.org host is back up again. Will use that then.

Reply:
Now installing linux-firmware from the checked out aports